main.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2016 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Main process for training classifiers."""
+
+# Preconfigure before starting main worker process.
+
+def configure():
+    import vm_config
```

Comment:
Yes we can. But we still need to execute vm_config.configure() before we start main worker process. One other way to do it is to modify PYTHONPATH from start.sh then we won't be needing this. I am confused between these two methods. Which one do you prefer?